packages/cxl-ui/src/components/cxl-notification-card.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import cxlNotificationCardStyles from '../styles/cxl-notification-card-css.js';
+
+@customElement('cxl-notification-card')
+export class CXLNotificationCardElement extends LitElement {
+  static get styles() {
+    return [cxlNotificationCardStyles];
+  }
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) theme = 'course';
```

Comment:
Does it need such property? If so, why is it course?

Reply:
I understood that those cards are based on courses and course can have different theme type like [here](https://feat-dashboard--conversionxl-aybolit.netlify.app/?path=/story/cxl-ui-cxl-course-card--cxl-course-card) (playbook, training etc). And this theme name will be like the tag on the card.

Reply:
Yes there will be courses, lessons, fast marketing and maybe other types of content that gets updated and therefore requires notification

Same with cards.

These exact themes still require definition. But for now for sure we have Courses, lessons and Fast marketing topics (simply **Topic** for now).

Reply:
So background color from design determines content type, it's not hover state @heshfekry ?

Reply:
Mathias improved Figma component and I made the necessary changes – hover, empty state and new container height logic. 

I'm confused about what are the Topics? In Figma there is Roadmap, All CXL updates as different tabs, are these the topics? And courses, that are the notification cards inside, those will have different background color based on theme (video, training etc)?